synchronizer/synchronizer_test.go

```diff
@@ -996,7 +1020,7 @@ func TestReorg(t *testing.T) {
 			m.State.
 				On("GetLastBlock", ctx, m.DbTx).
 				Return(lastBlock1, nil).
-				Once()
+				Maybe()
```

Comment:
same. You use it in many places. Please remove all maybe. You don't know what the code is doing using a maybe tag

Reply:
I have removed all unused Maybe() senteces